src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -7178,6 +7178,33 @@ async Task<int> goo()
             await VerifyItemWithMscorlib45Async(markup, "goo", description, "C#");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task AwaitableDotsLikeRangeExpression()
+        {
+            var markup = @"
+using System.IO;
+using System.Threading.Tasks;
+
+namespace N
+{
+    class C
+    {
+        Task M()
```

Comment:
I don't know if it matters for this test but shouldn't this method be `async` since it's using `await`?

Reply:
Thank you! Samples should not be 100% correct but I agree that adding async makes the test more accurate. Fixed.

---
In reply to: [335232868](https://github.com/dotnet/roslyn/pull/39302#discussion_r335232868) [](ancestors = 335232868)